execution-environment.yml

```diff
@@ -17,9 +17,11 @@ additional_build_files:
      dest: configs
 
```

Comment:
this `additional_build_files` line will fail because per [opcap-ansible-ee-build.yml](https://github.com/opdev/opcap-ansible/pull/103/commits/eb0f2e9c2557d544174e3e346ccabf98e2161286#diff-30f3ae4a5aeeeae924aa38e98e3cf94065fdd44689e7aeaba1593cc168f2aa02), file gets copied to `/home/runner/.ansible.cfg` and line 16 won't see it.

is there any particular reason we need to have the ansible.cfg baked into the image we end up pushing?

could we just set `ansible.cfg` as `ANSIBLE_CONFIG` var to ensure pull of private collections at ee build time?